docs/rules/no-unused-prop-types.md

```diff
@@ -86,12 +90,12 @@ AComponent.propTypes = {
   bProp: PropTypes.string // bProp is defined but never used
 };
 ```
-A suggested fix is to assign a bProp to a variable outside of the SFC.
```

Comment:
i prefer to keep the term SFC while it's still in common usage in the community. Please revert these.

Reply:
There's no mention of SFC in the react docs anymore. Or stateless function. https://www.dropbox.com/s/u51xabyw7wfuqqy/Screenshot%202019-05-12%2008.12.19.png?dl=0. And even if one does know the history and what SFC means, the warnings apply to function components with state (via Hooks) too.

Reply:
React docs are irrelevant; it’s still a widely used term in the community. A component with a useState hook is still an SFC.